.github/workflows/test-spec.yml

```diff
@@ -40,6 +40,8 @@ jobs:
             tests: plugins/streaming
           - version: 2.17.0
             tests: plugins/notifications
+          - version: 2.18.0
```

Comment:
2.18 hasn't been released so you need to have a `ref` and `hub` like below for a 2.18 build if you want to use that, but if query insights was added in 2.15 you should use 2.17, the latest stable release here.

Reply:
Got it. Just updated to 2.17. Thank you!